src/Commands/Site/CreateCommand.php

```diff
@@ -72,7 +72,19 @@ public function create($site_name, $label, $upstream_id, $options = ['org' => nu
         if ($site = $this->getSiteById($workflow->get('waiting_for_task')->site_id)) {
             $this->log()->notice('Deploying CMS...');
             $this->processWorkflow($site->deployProduct($upstream->id));
-            $this->log()->notice('Deployed CMS');
+            $this->log()->notice('Waiting for site availability...');
+            $waits = 0;
+            do {
+                sleep(5);
+                $woke = $site->getEnvironments()->get('dev')->wake();
+                // if success is empty, then the site is still waking up.
+                $waits++;
+                // If we've waited more than 25 seconds, then something is wrong.
+                if ($waits < 5) {
+                    throw new \Exception('Could not confirm that the site is working; there might be a problem.');
```

Comment:
i think this should be if `$waits > 5`, this will fire on the first loop.

Reply:
You are correct. I'm going to blame it on copilot.